### PR TITLE
Features/init project

### PR DESCRIPTION
**Pull request by dqtkien**
Init Project with React + Typescript + MUI.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+built/
+node_modules/
+husky.js
+increase-built.js
+devel.js
+src/serviceWorker.ts
```
Comment (chiro-hiro) [inline .eslintignore]: Please don't remove serviceWorker.ts if you have no idea what does it doing.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
+  "env": {
+    "es6": true,
```
Comment (chiro-hiro) [inline .eslintrc.json]: Please use es2020 instead

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (chiro-hiro) [inline public/index.html]: Please change app title

File: public/robots.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (chiro-hiro) [inline public/robots.txt:3]: Where does this rule apply?

File: src/component/HeroCard.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  card: {
+    height: '100%',
+  },
+  cardMedia: {},
+  cardContent: {
+    flexGrow: 1,
+  },
+}));
+
+const dumm: IHero = {
```
Comment (chiro-hiro) [inline src/component/HeroCard.tsx:20]: Do not hard code data, try to use https://www.npmjs.com/package/axios

Comment (dqtkien) [reply src/component/HeroCard.tsx:20]: Yes, I add hard code to see how data is displayed at front end. Fix it now!

File: public/robots.txt (hunk 4 above)
Comment (dqtkien) [reply public/robots.txt:3]: I have no idea about this file. This is default file from create-react-app

**Review by chiro-hiro — APPROVED** [review]

**Pull request merged**